rmw_connext_cpp/src/rmw_logging.cpp

```diff
@@ -0,0 +1,25 @@
+// Copyright 2014-2017 Open Source Robotics Foundation, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "rmw/rmw.h"
+
+extern "C"
+{
+// TODO(sriramster): Check support for log level manipulation with connext
+rmw_ret_t
+rmw_set_log_severity(rmw_log_severity_t severity)
+{
+  return RMW_RET_OK;
```

Comment:
As long as this is not implemented it shouldn't return "ok".

Reply:
Since, this is not implemented yet. An error returned might disturb the normal execution, so for time being till we find ways to implement the same inside DDS thought a "OK" (by-pass) would be better.

Reply:
Can you look into implementing this function or do propose to merge this as-is?

Reply:
@dirk-thomas Please merge as-is. I'll look into implementing this patch at later point. Thank you.